tests/Hamcrest/InvokedMatcherTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace Hamcrest;
+
+
+class SampleInvokeMatcher extends BaseMatcherTest
+{
+    private $matchAgainst;
+
+    public function __construct($matchAgainst)
+    {
+        $this->matchAgainst = $matchAgainst;
+    }
+
+    public function matches($item)
+    {
+        return $item == $this->matchAgainst;
+    }
+
+}
+
+class InvokedMatcherTest extends \PHPUnit_Framework_TestCase
+{
+    public function testInvokedMatchersCallMatches()
+    {
+        $sampleMatcher = new SampleInvokeMatcher("foo");
+
+        $this->assertEquals(
```

Comment:
1. use `assertTrue` and `assertFalse` (doing `assertEquals` on `true` is worse, because it can match `1` to `true`)
2. Please add assertion that would give non-matching value and check that a `false` was returned.